tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,112 @@
+            check_event = True
+
+    return check_event
+
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
+classes_subkey = os.path.join('SOFTWARE', 'Classes')
+
+sub_key_1 = os.path.join(classes_subkey, 'testkey')
+sub_key_2 = os.path.join(classes_subkey, 'Testkey')
+
+registry_1, registry_2 = os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+
+monitoring_modes = ['scheduled']
+
+# Configurations
+
+conf_params = {'WINDOWS_DUPLICATED_REGISTRY_1': registry_1,
+               'WINDOWS_DUPLICATED_REGISTRY_2': registry_2}
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_duplicated_registry.yaml')
+parameters, metadata = fim.generate_params(extra_params=conf_params, modes=monitoring_modes)
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+registry_list = [(key, sub_key_1, fim.KEY_WOW64_64KEY), (key, sub_key_2, fim.KEY_WOW64_64KEY)]
+daemons_handler_configuration = {'daemons': ['wazuh-syscheckd']}
+local_internal_options = {'syscheck.debug': '2', 'monitord.rotate_log': '0'}
+
+
+# Fixtures
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Test
+
+@pytest.mark.parametrize('key, subkey1, subkey2, arch', [(key, sub_key_1, sub_key_2, fim.KEY_WOW64_32KEY)])
+def test_registry_duplicated_entry(key, subkey1, subkey2, arch, get_configuration, configure_environment,
+                                   file_monitoring, configure_local_internal_options_module, daemons_handler,
+                                   wait_for_fim_start):
+    """Two registries with capital differences must trigger just one modify the event.
+
+    Test to check that two registries monitored with the same name but
+    capital differences only trigger one added event when the registry is created.
+
+
+    Params:
+        key (str): Name of the root subpath for registries.
+        subkey1 (str): Name of the subpath identifying the registry 1 (no capital letter in the name).
+        subkey2 (str): Name of the subpath identifying the registry 2 (capital letter in the name).
+        arch (str): Value holding the system architecture for registries.
+        get_configuration (fixture): Gets the current configuration of the test.
+        configure_environment (fixture): Configure the environment for the execution of the test.
+        restart_syscheckd (fixture): Restarts syscheck.
+        wait_for_fim_start (fixture): Waits until the first FIM scan is completed.
+
+    Raises:
+        TimeoutError: If an expected event (registry modified) couldn't be captured.
+    """
+
+    mode = get_configuration['metadata']['fim_mode']
+    scheduled = mode == 'scheduled'
+    monitorized_registry = os.path.join(key, subkey2)
```

Comment:
Same as before

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/c89eb367365e53d034a3f04058598377b3acc2c6)